Makefile

```diff
@@ -21,6 +21,11 @@ swarm:
 	@echo "Done building."
 	@echo "Run \"$(GOBIN)/swarm\" to launch swarm."
 
+ethkey:
```

Comment:
There is no need to add this command to Makefile. It can be built using `make all`.

Reply:
I removed it.